app/src/main/java/com/vero/tasky/agenda/domain/repository/AgendaRepository.kt

```diff
@@ -19,4 +19,6 @@ interface AgendaRepository {
     ) : Result<Unit>
 
     suspend fun getFullAgenda() : Result<Unit>
+
+    suspend fun logOut() : Result<Unit>
```

Comment:
This makes more sense to have it on the Authentication repository, since it's about the user's authentication status